spec/hashketball_spec.rb

```diff
@@ -60,22 +60,11 @@
   describe '#team_colors' do
 
     it 'knows the Brooklyn Nets colors are Black and White' do
-      brooklyn_colors = team_colors("Brooklyn Nets")
-      expect(brooklyn_colors.size).to eq(2)
-      brooklyn_colors.each do |color|
-        expect(["Black", "White"]).to include(color)
+      expect(team_colors("Brooklyn Nets")).to eq(["Black", "White"])
```

Comment:
Don't use `eq` for arrays. If they return `["White", "Black"]` this won't pass. Use `contain_exactly`